doc/dev/tutorial/juju-developer-tutorial.md

```diff
@@ -0,0 +1,125 @@
+# Juju developer tutorial
```

Comment:
This file doesn't need to be nested. It should be directly at main dev/tutorial.md.